libs/portals/admin/service-desk/src/hooks/useNationalId.tsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from 'react'
+
+interface UseNationalId {
+  nationalId: string
+  handleInputChange: (value: string) => void
+  isValid: boolean
+  formatNationalId: (value: string) => string
+}
+
+export const useNationalId = (): UseNationalId => {
+  const [nationalId, setNationalId] = useState('')
+  const [isValid, setIsValid] = useState(false)
+  const formatNationalId = (value: string) => {
+    if (value.length > 6) {
+      return `${value.slice(0, 6)}-${value.slice(6)}`.slice(0, 11)
+    } else {
```

Comment:
You created a formatNationalId util method that you should rather use.

Reply:
Was going to delete this after I created the formatNationalId, must have forgotten :D